projects/components/src/form/form-select/form-select.component.spec.ts

```diff
@@ -17,6 +17,11 @@ export class VcdFormSelectWidgetObject extends WidgetObject<FormSelectComponent>
         return this.findElement('select').nativeElement;
     }
 
+    get clrIcon(): string {
+        const clrIconDebugEl = this.findElement('clr-icon');
+        return clrIconDebugEl ? clrIconDebugEl.nativeElement.getAttribute('shape') : undefined;
```

Comment:
This can be simplified to

```ts
return clrIconDebugEl?.nativeElement.getAttribute('shape') || '';
```

We usually prefer to return `null` instead of `undefined`. In this case an empty string makes sense.

Reply:
Simplified the return statement.